ontology.ttl

```diff
@@ -93,6 +93,11 @@ foaf:logo rdf:type owl:AnnotationProperty .
 #    Object Properties
 #################################################################
 
+###  http://odahub.io/ontology#accessMode
+:accessMode rdf:type owl:ObjectProperty ;
+            rdfs:domain :StorageResource .
```

Comment:
Would be good to define `rdfs:range`.

May it be used in the definition of `:CRBeamS3` properties?

Reply:
What values are we going to use? String constants "read","read_write","read_append"?
What about type, should it be also DataProperty?

Reply:
If string values, then yes, it's a DataProperty and there is no need in range then